test/multi_target/multitarget_test.cpp

```diff
@@ -180,38 +180,79 @@ TEST_CASE(multitarget_compile_cpu_gpu)
     auto z_param = mm->add_parameter("z", s);
     auto cpu_ins = mm->add_instruction(
         migraphx::make_op("run_on_target", {{"target_id", 1}}), {x_param, y_param}, {cpu_mod});
+    auto cpu_ins_0 =
+        mm->add_instruction(migraphx::make_op("get_tuple_elem", {{"index", 0}}), cpu_ins);
     auto gpu_ins = mm->add_instruction(
-        migraphx::make_op("run_on_target", {{"target_id", 0}}), {cpu_ins, z_param}, {gpu_mod});
-    mm->add_return({gpu_ins});
-    p.compile({migraphx::make_target("gpu"), migraphx::make_target("cpu")});
+        migraphx::make_op("run_on_target", {{"target_id", 0}}), {cpu_ins_0, z_param}, {gpu_mod});
+    auto gpu_ins_0 =
+        mm->add_instruction(migraphx::make_op("get_tuple_elem", {{"index", 0}}), gpu_ins);
+
+    mm->add_return({gpu_ins_0});
+    migraphx::compile_options gpu_opts;
+    gpu_opts.offload_copy = true;
+    p.compile({migraphx::make_target("gpu"), migraphx::make_target("cpu")}, {gpu_opts});
     EXPECT(check_compiled_program(p, {migraphx::make_target("gpu"), migraphx::make_target("cpu")}));
+    migraphx::parameter_map params;
+    std::vector<float> x_data(s.elements(), 1);
+    std::vector<float> y_data(s.elements(), 2);
+    std::vector<float> z_data(s.elements(), 3);
+    params["x"] = migraphx::argument(s, x_data.data());
+    params["y"] = migraphx::argument(s, y_data.data());
+    params["z"] = migraphx::argument(s, z_data.data());
```

Comment:
You can use `fill_argument` to fill the buffer with a value.

Reply:
I had to change `fill_argument` to take `double` type as the value.